cmd/ksec/utils.go

```
@@ -13,3 +14,20 @@ func outputTabular(lines []string) {
 	}
 	w.Flush()
 }
+
+func askConfirmation(message string) bool {
+	fmt.Printf("%s [y/N]: ", message)
+
+	scanner := bufio.NewScanner(os.Stdin)
+	scanner.Scan()
+	response := scanner.Text()
+	if scanner.Err() != nil {
+		return false
```

This conditional seems redundant since default behavior is to return false and this isn't surfacing the error to the user. Should probably either surface the error or just not bother.

---

True, I'm showing the error and exiting now. Do you prefer it to return instead?